build/appveyor/before_build.bat

```diff
@@ -22,12 +22,12 @@ python build/add-mc-keys.py %MC_CONSUMER_KEY% %MC_CONSUMER_SECRET%
 )
 
 :: get revision number
-SET PATH=C:\Qt\5.4\mingw491_32\bin;C:\Qt\Tools\mingw491_32\bin;%PATH%
+SET PATH=C:\Qt\5.9.6\mingw53_32\bin;%PATH%

```

Comment:
`SET PATH=C:\Qt\5.9\mingw53_32\bin;C:\Qt\Tools\mingw530_32\bin;%PATH%`

Reply:
5.9.6, right?

Reply:
Grr, yes, of course